docs/book/src/images/cluster-topology-reconciller.puml

```diff
@@ -0,0 +1,50 @@
+@startuml
+'https://plantuml.com/component-diagram
+
+
+package "ClusterClass" as cc {
+  [Patches]
+  [Variable Schemas]
+  [Template 1]
+  [Template 2]
+  [\t...]
+
+  [Patches] -[hidden]-> [Variable Schemas]
+  [Variable Schemas] -[hidden]-> [Template 1]
+  [Template 1] -[hidden]-> [Template 2]
+  [Template 2] -[hidden]-> [\t...]
+}
+
+package "Cluster" as cluster {
+  [Variables]
```

Comment:
Maybe we should just call this point `Cluster` (the Variables are not an object, they are part of the Cluster resource and they are not reconciled)